velero/backup/common-service/label-bedrock-role.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: label-bedrock-job
+rules:
+  - verbs:
+      - get
+      - list
+      - update
+      - patch
+    apiGroups:
+      - ''
+      - extensions
+      - apps
+      - policy
+      - zen.cpd.ibm.com
+      - operators.coreos.com
+      - operator.ibm.com 
+    resources:
+      - zenservices
```

```suggestion
      - namespaces
      - zenservices
```

In `label-common-service.sh`, the ServiceAccount needs permission to get the namespace to pass the pre-requisite check

```
if ! $OC get namespace $OPERATOR_NS &>/dev/null; then
    error "Operator namespace $OPERATOR_NS does not exist, please provide a valid namespace"
fi
```